arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -1958,55 +1962,25 @@
       "properties": {
         "provisioningState": {
           "type": "string",
+          "readOnly": true,
           "description": "Provisioning state of the namespace."
         },
-        "status": {
```

Comment:
If the API version was returning status (duplicated), the API should continue to behave this way without any changes in the API version. Neither the specification nor the API should introduce a breaking change.

If this is only an error in the specification, then we should fix the specification. The breaking change is invalid as the field would never have worked in the first place, so anyone depending on this field would have already been broken.

If the change is in the API and not the spec, then we need to be careful with this change. The ARM API version is not a preview version moniker. To make a breaking change in an API, ARM requires a new API version be introduced.

@v-Ajnava which category does this change belong?